$MirOS: ports/devel/smpeg/patches/patch-Makefile_am,v 1.1 2006/03/23 23:09:40 tg Exp $
$OpenBSD: patch-Makefile_in,v 1.2 2001/06/28 00:56:58 pvalchev Exp $
--- Makefile.am.orig	Wed Apr  4 20:27:52 2001
+++ Makefile.am	Sun Feb  1 11:05:46 2009
@@ -33,7 +33,6 @@ libsmpeginclude_HEADERS =	\
 	smpeg.h
 
 libsmpeg_la_LDFLAGS = 		\
-        -release $(LT_RELEASE)	\
 	-version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE)
 
 libsmpeg_la_LIBADD =		\
@@ -53,11 +52,7 @@ EXTRA_DIST =		\
 	autogen.sh
 
 # Sample MPEG players
-if HAVE_GTK
-GTK_PLAYER = gtv
-else
 GTK_PLAYER =
-endif
 if HAVE_OPENGL
 OPENGL_PLAYER = glmovie
 else
@@ -69,10 +64,6 @@ bin_PROGRAMS = plaympeg $(GTK_PLAYER) $(
 man_MANS = plaympeg.1 gtv.1
 plaympeg_SOURCES = plaympeg.c
 plaympeg_LDADD = libsmpeg.la
-
-# Sources for gtv
-gtv_SOURCES = gtv.c gtv.h
-gtv_LDADD = @GTK_LIBS@ libsmpeg.la
 
 # Sources for glmovie
 glmovie_SOURCES = glmovie-tile.c glmovie.c glmovie.h
